commit 63974b4364460425bf3f7795b4e1114d5f2cb6ca (refs/changes/95/495/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-09-17T17:49:35-07:00 (3 years, 1 month ago)
    
    joyent/node-triton#131 update 'make cutarelease' and 'make versioncheck' to avoid commits for cutting a release

diff --git a/Makefile b/Makefile
index 6951165..b74b5b4 100644
--- a/Makefile
+++ b/Makefile
@@ -69,7 +69,7 @@ cutarelease: versioncheck
 	    date=$(shell date -u "+%Y-%m-%d") && \
 	    git tag -a "$$ver" -m "version $$ver ($$date)" && \
 	    git push --tags origin && \
-	    npm publish $$name
+	    npm publish
 
 .PHONY: git-hooks
 git-hooks:
